src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -150,7 +150,7 @@ public static Set<Role> parseRoles(Collection<String> roles) throws ParseExcepti
     public static Profile parseProfileName(String profileName) throws ParseException {
         requireNonNull(profileName);
         String trimmedProfileName = profileName.trim();
-        if (!Profile.isValidProfile(trimmedProfileName)) {
+        if (!Profile.isValidProfileName(trimmedProfileName)) {
             throw new ParseException(Profile.MESSAGE_CONSTRAINTS);
```

Comment:
I might prefer `throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, Profile.MESSAGE_CONSTRAINTS));`
basically also use the "Invalid  command format!" message before the constraint message

Reply:
Thanks. ive updated the error messages and help window